pipelines/release.yml

```diff
@@ -1965,131 +2024,257 @@ resources:
   source:
     repository: cloudfoundry-incubator/backup-and-restore-sdk-release
 
-- name: mock-resource
-  type: github-release
+- name: mock-resource-alpine
+  type: registry-image
   icon: *release-icon
   source:
-    owner: concourse
-    repository: mock-resource
-    access_token: ((concourse_github_dummy.access_token))
-    semver_constraint: ((resource_type_versions.mock))
+    repository: concourse/mock-resource
+    tag: ((resource_type_versions.mock))-alpine
+    username: ((docker.username))
```

Comment:
Do we need to give it a username and password if we are just pulling from docker hub?

Reply:
Only reason I added it was to avoid any rate limit issues since this means we're pulling a lot more images. Based on https://www.docker.com/increase-rate-limits it goes from 100 to 200 for auth'd requests.